grakn-core/src/main/java/ai/grakn/util/REST.java

```diff
@@ -46,6 +46,7 @@
          */
         public static class Tasks {
             public static final String TASKS = "/tasks";
+            public static final String TASKS_BULK = "/tasks/bulk";
```

Comment:
I don't think there should be a separate end point for bulk loading. Unless there is a clear advantage I am missing. We only have one pipeline for processing tasks so there should only be one endpoint for uploading those tasks.